packages/@lwc/module-resolver/src/index.ts

```diff
@@ -73,7 +100,10 @@ function resolveModulesFromNpm(packageName: string): RegistryEntry[] {
         } else {
```

Comment:
Github won't let me comment on that line, but on line 72 you are passing in `{ rootDir: lwcConfigFile }` and it should be the config file directory path, not the config file path.